src/main/webapp/room.js

```diff
@@ -0,0 +1,19 @@
+let players = [];
+
+const createUser = (ev)=>{
+    ev.preventDefault();  //to stop the form submitting
+    let user = {
+        username: document.getElementById('new-player-id').value,
+        get userID() {
```

Comment:
This should be "userId" and not "userID", similar to how it's "getElementById" above and not "getElementByID"

You can see a similar suggestion in the JavaScript style guide below:
https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers